pycontrails/models/cocip/cocip.py

```diff
@@ -596,17 +595,20 @@ def _process_emissions(self) -> None:
         """
         logger.debug("Processing flight emissions")
 
-        # currently only one emissions model
-        emissions = Emissions(copy_source=False)
+        # Call aircraft performance and Emissions models as needed
+        # NOTE: None of these sub-models actually do any met interpolation -- self.source already
+        # has all of the required met variables attached. Therefore, we don't need to worry about
+        # being consistent with passing in Cocip's interp_kwargs and humidity_scaling into
+        # the sub-models.
+        emissions = Emissions()
+        ap_model = self.params["aircraft_performance"]
```

Comment:
Would it be too verbose to call this `performance_model` or `aircraft_performance` just to make it clear throughout?

Reply:
I see this is consistent with `cocip_grid` - Fine leaving it as it

Reply:
👍